### ISSUE TITLE
Update theme color for accessibility

### ISSUE THREAD

**Issue by clairesunstudio** [massgov/mayflower#154]
I think we should make our primary theme colors more accessible in general and have a more flexible usage. Currently the tell us what you think feedback button is failing accessibility check. Proposing to update the green theme color everywhere, instead of just updating it in one place:
before:
<img width="796" alt="screen shot 2018-08-09 at 5 23 19 pm" src="https://user-images.githubusercontent.com/5789411/43926804-6220c07e-9bf9-11e8-80b2-877490c08205.png">
after:
![screen shot 2018-08-10 at 10 16 42 am](https://user-images.githubusercontent.com/5789411/43962822-aa46b57e-9c86-11e8-91b8-f71900044d65.png)

**Comment by clairesunstudio** [comment]
#155
